components/automate-gateway/handler/iam/v2beta/users/users.go

```diff
@@ -43,8 +43,8 @@ func (p *Server) CreateUser(
 	return &pb_resp.CreateUserResp{User: convert(resp)}, nil
 }
 
-func (p *Server) GetUsers(
-	ctx context.Context, _ *pb_req.GetUsersReq) (*pb_resp.GetUsersResp, error) {
+func (p *Server) ListUsers(
+	ctx context.Context, _ *pb_req.ListUsersReq) (*pb_resp.ListUsersResp, error) {
 	resp, err := p.users.GetUsers(ctx, &local_user.GetUsersReq{})
```

Comment:
If consistency is the main goal here, I think we should update the method names on the domain level as well (which you already did with teams)

Reply:
i spose you've kept GetUsers here since it's used by the v1 API. that works for me, don't want to blow this up into a huge rename